File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverHere/Context.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Context {

    @SerializedName("location")
    @Expose
    private Location location;

    public Location getLocation() {
        return location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/Context.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere

import com.google.gson.annotations.Expose

class Context {
    @SerializedName("location")
    @Expose
    private var location: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.Location? = null
    fun getLocation(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.Location? {
        return location
    }

    fun setLocation(location: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.Location?) {
        this.location = location
    }
}